### PR TITLE
Add missing mag I2C bus define to JHEF435

### PR DESCRIPTION
**Pull request by ot0tot**

### CODE REVIEW

File: configs/JHEF435/config.h [hunk 1]
```diff
@@ -97,4 +97,5 @@
 #define MAX7456_SPI_INSTANCE    SPI2
 #define FLASH_SPI_INSTANCE      SPI3
 #define BARO_I2C_INSTANCE       (I2CDEV_2)
+#define MAG_I2C_INSTANCE        (I2CDEV_2)
```
Comment (KarateBrot) [inline configs/JHEF435/config.h:100]: No idea why everyone puts braces around `I2CDEV_X`
```suggestion
#define MAG_I2C_INSTANCE        I2CDEV_2
```

**Review by KarateBrot — APPROVED** [review]

File: configs/JHEF435/config.h (hunk 1 above)
Comment (ot0tot) [reply configs/JHEF435/config.h:100]: I don't think it makes a difference. We can clean this up and standardize in a future PR.

**Comment by ot0tot** [comment]
Please review this comprehensive PR instead that fixes this issue in all targets: https://github.com/betaflight/config/pull/580

**Comment by haslinghuis** [comment]
Resolved in #580

**Pull request closed without merging**